src/Core/Repositories/EntityFramework/Queries/UserCollectionDetailsQuery.cs

```diff
@@ -0,0 +1,57 @@
+using System.Linq;
+using Bit.Core.Models.Table;
+using System;
+using Bit.Core.Enums;
+using System.Collections.Generic;
+using Core.Models.Data;
+using Bit.Core.Models.Data;
+
+namespace Bit.Core.Repositories.EntityFramework.Queries
+{
+    public class UserCollectionDetailsQuery : IQuery<CollectionDetails>
+    {
+        private Guid? UserId { get; set; }
```

Comment:
So in this pattern, not sure how the `Query` thing may work, but from appearances I would recommend:
* Make `UserId` instead a `private readonly Guid? _userId;` (remove the `get;set;` property accessors)
* Assign it per usual in the constructor

Reply:
Changed on all queries